### PR TITLE
The Liberation is at Hand

### PR DESCRIPTION
**Pull request by Kurfursten**
A new ammolathe option costing 8000 iron, 6000 glass, 2000 plastic. The "liberator" laser carbine was designed to fill the energy-equivalent niche of the hunting rifle on frontier worlds, ideal for hunting game. Its modular lasers disperse on impact with human skin, lowering the risk of rebellion. They intensify in contact with the roiling spore surface of a blob however...

* Large like a musket, fits on back or belt. Do not need to wield. Cannot take a scope.
* 10 damage vs humans; 40 damage vs blobs; 25 damage vs other targets
* Fires six times before recharging (60 per shot, 250 max charge)

When damaging a blob, extra damage can "splash" to a nearby blob piece, meaning one shot can take out two unupgraded blob pieces.

🆑 
* rscadd: The liberator anti-blob rifle is now available at the ammolathe. It deals severely reduced damage to humans, so it's a lower risk option to distribute to civilians.

### CODE REVIEW

**Comment by cloutcode** [comment]
what does it look like

**Comment by Kurfursten** [comment]
Needs spritework

**Comment by boy2mantwicethefam** [comment]
I'm going to support the PR now and get mad later when I get mowed down by the massed volley fire of crewmembers rebelling against the british blobccupation

**Comment by Servalfang** [comment]
maybe undo some blob nerfs if this gets merged? Sounds extremely powerful.

**Comment by Kurfursten** [comment]
The liberator shines against areas the blob isn't paying attention - particularly unupgraded regular blobs. Against strong blobs (which as you know are EVERYWHERE the blob is looking), laser cannons remain stronger in every way.

VS STRONG BLOB
| Liberator  | Laser Cannon |
| ------------- | ------------- |
| 4 shots to kill  | 3 shots to kill  |
| 6 shots max  | 10 shots max |
| large size | medium size |

Like laser cannons, you need to manufacture liberators, but you don't need to do research or get asteroid materials for them. I believe they will not swing the balance away from the blob on their own and my testing so far supports this.

In short, **I would not be concerned about need for a blob buff at this time.** But we should of course always keep an eye on it.

**Comment by Dacendeth** [comment]
Cool, gives blob another target to explode at for a change in gameplay.

**Comment by Kurfursten** [comment]
Added icon courtesy of Blithering

Also lowered the charging time to offset the lower shot count, did some more testing. Feels great vs. unupgraded blobs but strong blobs remain tough to crack.

**Comment by SonixApache** [comment]
idea: change the laser projectile sprite into the emitter beam, maybe it'll help people realize that it's a blobkiller rifle

**Comment by Kurfursten** [comment]
I think that idea is interesting but I'm not sure it's ideal. The carbine does make a great initial add for blob defense but like I said, it isn't the end game for it. It can also be useful in other situations like snaxi, spiders or away missions to avoid arming the crew with deadly lasers.

**Comment by SonixApache** [comment]
giving it the thinner-than-normal white laser beam sprite might make people understand that it's not a normal laser gun
